types/react-hyperscript-helpers/react-hyperscript-helpers.types.ts

```diff
@@ -25,6 +25,9 @@ interface TestComponentProps {
 
 const TestComponent = (props: PropsWithChildren<TestComponentProps>) => div()
 
+// PropsWithChildren with key
+h(TestComponent, { key: 'key', stringProp: 'value' })
+
```

Comment:
How did the line below pass before, because it didn't have `key` spelled out?

```typescript
h(TestComponent, { stringProp: 'value', optionalNumberProp: 1 })
```

Reply:
Right. The problem is `h` won't accept a `key`. Currently, no line in this test uses a `key`.